regression/cbmc/union11/union_list.c

```diff
@@ -0,0 +1,30 @@
+#include <assert.h>
+
+struct list_item
+{
+  struct list_item *previous;
+};
+
+struct List
+{
+  unsigned something;
```

Comment:
Unused in the test case? Is it important that neither struct member is at zero offset? If so comment inline about why

Reply:
It's crucial to make this pass/fail. I have added both an inline comment and information to the commit message.